scripts/backup_outputs.py

```
@@ -0,0 +1,42 @@
+
+# Add it to sys.path
+if current_dir not in sys.path:
+    sys.path.append(current_dir)
+
+current_dir = Path(os.getcwd())
+parent_dir = Path(os.path.realpath(__file__)).parent.parent
+
+if current_dir != parent_dir:
+    raise Exception(f"Script can only be run in {parent_dir}")
+
+def backup_outputs():
```

Ruff (0.12.2)

Missing return type annotation for public function `backup_outputs`

Add return type annotation: `None`

(ANN201)